### PR TITLE
Align docs/ui.md with www/README.md

### PR DESCRIPTION
**Pull request by jackgr**
This PR contains changes to docs/ui.md and www/README.md to bring docs/ui.md up to date with #9000, and to align the contents of the two files.

cc @bgrant0607, @bcbroussard, @preillyme

### CODE REVIEW

File: docs/ui.md [hunk 1]
```diff
@@ -1,46 +1,20 @@
...
-
-To rebuild the UI, run the following:
-
-```sh
-hack/build-ui.sh
-```
+You can also use other web servers to serve the contents of the www/app directory, as described [here](../www/README.md#serving-the-app-during-development). 
 
+### Running remotely
+When Kubernetes is deployed remotely, the api server deploys the UI. To access it, visit `/static/app/` on your master server.
```
Comment (bgrant0607) [inline docs/ui.md]: /ui now should redirect to the ui.

Comment (jackgr) [inline docs/ui.md]: Indeed, it does. Nice. Updated the docs accordingly.

**Comment by jackgr** [comment]
@bcbroussard Not sure which sections of which document your comments address. Can you please clarify?

**Comment by bcbroussard** [comment]
@jackgr I'm not sure why the code snippets went away, but you can see the comments with code here https://github.com/GoogleCloudPlatform/kubernetes/commit/97b1866175ab22116c18549e574563cb1531d7a9

**Comment by jackgr** [comment]
@bcbroussard Agree with your point that many people will come here first. However, this document is about working the UI, not about accessing and using it. We have a document about accessing and using the UI, which is `docs/ui.md`. 

Moving the section in question to the top would move it out of the parent section about building and deploying the UI, and break the development/production symmetry of that section. 

Rather than try to make this document do double duty, I've added two lines at the top, pointing the reader to `docs/ui.md` for information on accessing and using the UI. This seems a clean solution, since readers who arrive at `docs/ui.md` looking for information about working with the UI will likewise find a pointer to this document.

In summary, we have two documents about the UI: `docs/ui.md` explains how to access and use it, while `www/README.md` explains how to work with it (i.e., how to configure, build, deploy and update it).

**Comment by bcbroussard** [comment]
Thanks @jackgr, adding those lines at the top works for me. 

What about removing the proxy instructions from `docs/ui.md`?

**Comment by jackgr** [comment]
Per my previous comment, the proxy is still a valid way to serve the UI, and people familiar with the old UI will be expecting it. Seems reasonable to keep the proxy instructions, showing how to serve the new UI, and giving them insight, via the pointer to www/README.md, into alternate ways to serve the new UI. In other words, I don't see a good motivation to remove the proxy instructions.

**Comment by bgrant0607** [comment]
Good enough for now. LGTM.
